docs/pim/products.md

```diff
@@ -21,6 +21,14 @@ can store different types of information.
 For example, products of publication type contain different Fields than white 
 appliances.
 
+You can choose between two available types: `Physical` and `Virtual`:
+
+- `Physical` - tangible products with assigned stock. 
```

Comment:
```suggestion
- Physical - tangible products with assigned stock. 
```
Physical product don't always have assigned stock, the stock can be infinite. Well, strictly speaking infinite stock is still stock, but this can be confusing.